pkg/render/node.go

```diff
@@ -143,6 +147,20 @@ func (c *nodeComponent) ResolveImages(is *operatorv1.ImageSet) error {
 		errMsgs = append(errMsgs, err.Error())
 	}
 
+	if vppDataplaneEnabled(c.cfg.Installation) {
+		c.vppImage, err = components.GetReference(components.ComponentCalicoVpp, reg, path, prefix, is)
+	}
+	if err != nil {
```

Comment:
This error check should be within the `vppDataplaneEnabled` branch above where the error is generated